float64[12] joint_q

